src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -357,21 +358,51 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
 
   render() {
     const { artwork, inquiryComponent } = this.props
+    const isPriceListed = !artwork?.isPriceHidden
+    const isOfferableFromInquiry = artwork?.isOfferableFromInquiry
+    const isOfferable = artwork?.is_offerable
+    const isAcquireable = artwork?.is_acquireable
+    const isInquireable = artwork?.is_inquireable
+    const labFeatureEnabled = userHasLabFeature(
+      this.props.user,
+      "Make Offer On All Eligible Artworks"
+    )
 
     const {
       isCommittingCreateOrderMutation,
       isCommittingCreateOfferOrderMutation,
       selectedEditionSet,
     } = this.state
 
-    const artworkEcommerceAvailable = !!(
-      artwork.is_acquireable || artwork.is_offerable
-    )
+    const artworkEcommerceAvailable = !!(isAcquireable || isOfferable)
 
-    if (!artwork.sale_message && !artwork.is_inquireable) {
+    if (!artwork.sale_message && !isInquireable) {
       return <Separator />
     }
 
+    const shouldDisplayMakeOfferButton = (): boolean => {
+      if (
+        isOfferable ||
+        (isPriceListed && isOfferableFromInquiry && labFeatureEnabled)
+      ) {
+        return true
+      }
+      return false
+    }
+
+    const shouldDisplayMakeOfferAsPrimary = (): boolean => {
```

Comment:
I think all the "shouldDisplayX" functions could simply be constants instead, right?